modules/programs/lazygit.nix

```diff
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.lazygit;
+
+  yamlFormat = pkgs.formats.yaml { };
+
+  inherit (pkgs.stdenv.hostPlatform) isDarwin;
+
+in {
+  meta.maintainers = [ maintainers.kalhauge ];
+
+  options.programs.lazygit = {
+    enable = mkEnableOption "simple terminal UI for git commands";
+
+    settings = mkOption {
+      type = yamlFormat.type;
+      default = { };
```

Comment:
```suggestion
      default = { };
      defaultText = literalExample "{ }";
```

Reply:
Reading through the documentation this does not look like its strictly necessary. Is it a code-style thing?